community-membership.md

```diff
@@ -1,77 +1,198 @@
+
+- Approver status may be a precondition to accepting large code contributions
+- Demonstrate sound technical judgement
+- Responsible for project quality control via [code reviews](contributors/devel/collab.md)
+  - Focus on holistic acceptance of contribution such as dependencies with other features, backwards / forwards compatibility, API and flag definitions, etc
+- Expected to be responsive to review requests as per [community expectations](contributors/devel/community-expectations.md);
+- Mentor contributors and reviewers
+- May approve code contributions for acceptance
+
+## Owner
+
+Owners of an area / component are approvers of an entire area that have demonstrated good judgement and
+responsibility.  Owners accept design proposals and approve design decisions for their area of ownership.
+
+**Defined by:** *approver* entry in the top-level OWNERS file for the area in the kubernetes codebase
+
+### Requirements
+
+The following apply to the area / component for which one would be an owner.
+
+- Originally authored or contributed major functionality to an area
+- An approver in the **top-level** [OWNERS](contributors/devel/owners.md) files
+
+### Responsibilities and privileges
+
+The following apply to the area / component for which one would be an owner.
+
+- Owner status may be a precondition to accepting a new component or piece of major functionality
+- Design/proposal approval authority over the area / component, though escalation to kubernetes-maintainers is possible.
+- Mentor and guide approvers, reviewers, and members.
+
+## Maintainer
+
+[**Kubernetes project maintainers**](https://github.com/orgs/kubernetes/teams/kubernetes-maintainers) work holistically
+across the project to maintain its health and success.  They are typically involved in multiple different areas, and
+have made sustantial contributions both through code and broader organizational efforts.
+
+**Defined by:** *write* access to the kubernetes GitHub repo
+
+TODO: Determine if this role is outdated and needs to be redefined or merged into owner role.
+
+### Requirements
+
+- Approver for some part of the codebase for at least 3 months
+- Member for at least 1 year
+- Primary reviewer for 20 substantial PRs
+- Reviewed or merged at least 50 PRs
+- Apply to `kubernetes-maintainers@googlegroups.com`, with:
+  - A [Champion](https://github.com/kubernetes/community/blob/master/incubator.md#faq) from the existing
+    kubernetes-maintainers members
+  - A Sponsor from Project Approvers
+  - Summary of contributions to the project
+  - Current project responsibilities
+  - Links to merged and assigned PRs
+- At least 3 of the maintainers must approve the application, with no objections
+- Application expires after 2 weeks if not enough approvals are granted
+
+### Responsibilities and privileges
+
+- Write access to repo (assign issues/PRs, add/remove labels and milestones, edit issues and PRs, edit wiki, create/delete labels and milestones)
```

Comment:
I think we would need to for reviewer, so that the person can join the `kubernetes-reviewers` team.